pkg/apis/admission/BUILD

```diff
@@ -0,0 +1,46 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Yeah - as @smarterclayton wrote - you don't need to split it by type. It's fine to have all auto-generated things in a single commit.